api/process.go

```diff
@@ -52,10 +52,7 @@ func (s *service) Start(ctx context.Context, args *task.StartArgs) (*task.StartR
 	if args.JID == "" {
 		state.JID = xid.New().String()
 	} else {
-		existingState, err := s.getState(args.JID)
-		if err != nil {
-			return nil, status.Error(codes.NotFound, "failed to check existing jobs")
-		}
```

Comment:
Bolt db returned a nil err if no row is found in the table. sqlite db returns a non-nil err. If that check is kept, it will always fail (which also caused the smoke tests to fail previously)